docs/UserGuide.md

```diff
@@ -62,111 +74,128 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
+
+### Deleting entries : `del INDEX_OF_ENTRY t/ENTRY_TYPE`
+Deletes an entry.
+1. Example: `del 2 t/e` deletes the 2nd item on the expenditure list 
+   Expenditure list:
+   1. Movie 15.60 12 Sep 2022 c/Entertainment
+   2. Drink 1.20 12 Sep 2022 c/Food 
+   Expected: `Deleted Entry: Drink; Date: 12-09-2022; Amount: 1.20; Tag: Food`
+   
+1. Example: `del 2 t/i` deletes the 2nd item on the income list
+   Income list:
+   1. Tuition 40.00 12 Sep 2022 c/Salary
+   2. Allowance 100.00 12 Sep 2022 c/Allowance
+   Expected: `Deleted Entry: Allowance; Date: 12-09-2022; Amount: 100.00 Tag: Allowance`
+
+
+### Editing entries : `edit INDEX_OF_ENTRY t/ENTRY_TYPE [d/EDITED_DESCRIPTION a/EDITED_AMOUNT da/EDITED_DATE c/EDITED_CATEGORY]`
+Edits an entry, at least **1** of the optional fields must be present.
+
+1. Edits expenditure with specified fields
+   * Example: `edit 1 t/e d/ChickenRice`
+   * The expenditure at index 1 will have its description edited
+   * Expected: `Edited Entry: ChickenRice; Date: 21-10-2022; Amount: 4.20; Tag: Food`
+1. Edits income with specified fields
+   * Example: `edit 2 t/i a/150.00 da/22-10-2022`
+   * The income at index 2 will have its amount and date edited
+   * Expected: `Edited Entry: CafeSalary; Date: 22-10-2022; Amount: 150.00; Tag: Salary`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 The default date is the current date on your computer!
 </div>
 
-### Deleting income: `del -i NUMBER INDEX_OF_INCOME`
-
-Deletes an income entry
-
-Example: `del -i 2` deletes the 2nd item on the list
-
-Spending list:
-1. Earned 5.00 12 Sep 2022
-2. Earned 30.00 12 Sep 2022
-
-Expected: `An income of 30.00 on 12 Sep 2022 has been deleted.`
-
+### Summary of entries : `summary [mo/MONTH]`
 
-### Viewing total summary of spendings : `view -s [MONTH]`
+1. View a summary of all entires
+   * Examples: `summary`
+     * Expected: <br/>
+       Financials Summarized <br/>
+       Total Expenditure: 154.40 <br/>
+       Total Income: 150.00 <br/>
+       Total Balance: -4.40 <br/>
+2. View a summary of all entires the specified month
+    * Examples: `summary mo/2022-09`
+    * Expected: A summary of entries in September 2022: `Total amount spent in September 2022: $37.70`
 
-1. View a summary of all spending
-   * Examples: `view -s`
-   * Expected: A summary of total spending: `Total amount spent: $125.30`
-2. View a summary of all spending the specified month
-    * Examples: `view -s 09-2022`
-    * Expected: A summary of spendings in September 2022: `Total amount spent in September 2022: $37.70`
-
-* View all spending recorded by the user in a month. The month refers to the month that is displayed to the user.
+* Provides a financial summary recorded by the user in a month. The month refers to the month that is displayed to the user.
 * The `MONTH` field is optional, if no month is specified, the application displays the current month.
```

Comment:
I think this might be confusing as current if no month is specified all the entries is summarised

Reply:
have changed this to:
if no month is specified, the application displays the summary for all entries.